source/community/release_notes/teak/dev_op_release_notes.rst

```diff
@@ -294,6 +294,84 @@ Removed Settings
 Other Operator Changes
 ======================
 
+- In LMS and CMS, Celery now uses task protocol 2.
+   - *Action*: Any operator using custom Celery tooling should ensure it is compatible with protocol 2. For other operators, no action is required.
+   - *Background*: Celery 4.0 `switched how task messages are structured <https://docs.celeryq.dev/en/stable/history/whatsnew-4.0.html#new-protocol-highlights>`_ and
+     the new message format is called protocol 2. The version of Celery we currently use (anything >=4.0) can create and consume both protocol versions and it should
+     be safe to switch between them with zero downtime.
+
+      - By default, Celery 4.0 and higher produce messages in this format, and Celery 3.1.25 and higher can read messages in this format.
+      - edx-platform was pinned to protocol 1 `during the upgrade <https://github.com/openedx/edx-platform/pull/24916>`_ to Celery 4, presumably as a precaution. 
+        This change is the long-delayed unpinning of the protocol version so that Celery can use its default version.
+      - Operators can still override the protocol version using the Django setting `CELERY_TASK_PROTOCOL` although there is no guarantee that protocol 1 compatibility
```

Comment:
For code things (file paths, settings, flags) it'd be great to render these in code formatting, with two backticks:

```suggestion
      - Operators can still override the protocol version using the Django setting ``CELERY_TASK_PROTOCOL`` although there is no guarantee that protocol 1 compatibility
```

Not going to flag all of them but if you could go through and find them that'd be great (that includes things like import matlib, ./tmp/, etc below